include/class.mailer.php

```diff
@@ -514,7 +514,7 @@ function($match) use ($domain, $mime, $self) {
             // Force reconnect on next ->send()
             unset($smtp_connections[$key]);
 
-            $alert=sprintf(__("Unable to email via SMTP:%1\$s:%2\$d [%3\$s]\n\n%4\$s\n"),
+            $alert=sprintf(_S("Unable to email via SMTP:%1\$s:%2\$d [%3\$s]\n\n%4\$s\n"),
```

Comment:
Could we change this to
```php
$alert=_S("Unable to email via SMTP")
    . sprintf(":%1\$s:%2\$d [%3\$s]\n\n%4\$s\n",
```
So that the translators don't have to deals with the string of random token and newline chars?